onnx/optimizer/passes/fuse_add_bias_into_conv.h

```diff
@@ -30,11 +30,16 @@ struct FuseAddBiasIntoConv final : public OptimizePass {
           auto orig_bias = n->inputs()[1 - idx];
           auto conv_shape = orig_conv->sizes();
           auto bias_shape = orig_bias->sizes();
-          if (bias_shape.size() == 0 || conv_shape.size() == 0) {
+          auto weight_shape = orig_conv->node()->inputs()[1]->sizes();
+          if (bias_shape.size() == 0
+              || conv_shape.size() == 0
```

Comment:
Here should be (conv_shape.size() == 0 && weight_shape.size() == 0).

Either we have conv_shape[1] or weight_shape[1], we can proceed. Because they are equivalent.